cobra/flux_analysis/deletion.py

```diff
@@ -218,153 +183,182 @@ def _element_lists(entities, *ids):
-        This must be a QP-capable solver for MOMA. If left unspecified,
-        a suitable solver will be automatically chosen.
+    method: {"fba", "moma", "linear-moma"}, optional
+        Method used to predict the growth rate.
 
-    zero_cutoff: float, optional
-        When checking to see if a value is 0, this threshold is used.
+    num_jobs : int, optional
+        The number of parallel processes to run. Can speed up the computations
+        if the number of knockouts to perform is large. If not passed,
+        will be set to the number of CPUs found.
 
     Returns
     -------
-    dict of dict
-        A sparse representation of all combinations of pairs of reaction
-        deletions without replacements.
+    pandas.DataFrame
+        A representation of all combinations of reaction deletions. The
+        columns are 'growth' and 'status', where
+
+        index : frozenset([str])
+            The reaction identifiers that were knocked out.
+        growth : float
+            The growth rate of the adjusted model.
+        status : str
+            The solution's status.
+
     """
 
     reaction_list1, reaction_list2 = _element_lists(model.reactions,
                                                     reaction_list1,
                                                     reaction_list2)
-    return _multi_deletion(model, 'reaction',
-                           element_lists=[reaction_list1, reaction_list2],
-                           **kwargs)
-
+    return _multi_deletion(
+        model, 'reaction', element_lists=[reaction_list1, reaction_list2],
+        method=method, num_jobs=num_jobs)
 
-def double_gene_deletion(model, gene_list1=None, gene_list2=None, **kwargs):
-    """Sequentially knocks out pairs of genes in a model
 
-    model : :class:`~cobra.core.Model.Model`
-        cobra model in which to perform deletions
+def double_gene_deletion(model, gene_list1=None, gene_list2=None,
+                         method="fba", num_jobs=None):
+    """
+    Knock out each gene pair from two given lists.
 
-    gene_list1 : [:class:`~cobra.core.Gene.Gene`:] (or their id's)
-        Genes to be deleted. These will be the keys in the resulting dict.
-        If not provided, all genes will be used.
+    Parameters
+    ----------
+    model : cobra.Model
+        The metabolic model to perform deletions in.
 
-    gene_list1 : [:class:`~cobra.core.Gene.Gene`:] (or their id's)
-        Genes to be deleted. These will be the keys in the resulting dict.
-        If not provided, gene_list1 will be used.
+    gene_list1 : iterable, optional
+        First iterable of ``cobra.Gene``s to be deleted. If not passed,
+        all the genes from the model are used.
 
-    method: "fba" or "moma"
-        Procedure used to predict the growth rate
+    gene_list2 : iterable, optional
+        Second iterable of ``cobra.Gene``s to be deleted. If not passed,
+        all the genes from the model are used. The order independent
```

Comment:
maybe "ordered"?

Reply:
"set of gene combinations" maybe? I wanted to emphasize that they are order-independent but maybe I should stop nit-picking.